Krivorotova Tatyana/grid_layout/grid.css

```diff
@@ -0,0 +1,137 @@
+body{
```

Comment:
![grid-layout-tatyana](https://user-images.githubusercontent.com/15708576/61233601-9073e680-a739-11e9-97bd-d9011d7d6fa6.png)
There is a little problem with layout on some viewports